TGC/decisions/FIL.txt

```diff
@@ -0,0 +1,125 @@
+political_decisions = {
+
+
+### Filibusters ###
+
+	## William Walker's Filibuster state wins civil war, gets cores on Central America
+	FIL_manifest_destiny = {
+		picture = conquest_of_the_south
+		potential = {
+			tag = FIL
+			NOT = { has_country_flag = fil_hasmanifestdestiny }
+			has_country_flag = fil_declare_republic
+		}
+
+		allow = {
+			war = no
+			war_exhaustion = 0
+		}
+
+		effect = {
+			prestige = 10
+			set_country_flag = fil_hasmanifestdestiny
+			badboy = 20
+
+			## Cores ##
+				UCA_2186 = { add_core = FIL } # Guatemala
+        		ENG_2187 = { add_core = FIL } # Belize (British Honduras)
+        		UCA_2191 = { add_core = FIL } # El Salvador
+        		UCA_2193 = { add_core = FIL } # Honduras
+        		UCA_2197 = { add_core = FIL } # Nicaragua
+        		UCA_2201 = { add_core = FIL } # Costa Rica
+				
+			## Diplomacy ##
+				relation = { who = ENG value = -75 }
+				leave_alliance = ENG
+				COS = {
+					country_event = {
+						id = 8870786
+						days = 28
+					}
+				}
+				HON = {
+					country_event = {
+						id = 8870786
+						days = 141
+					}
+				}
+				GUA = {
+					country_event = {
+						id = 8870786
+						days = 141
+					}
+				}
+				ELS = {
+					country_event = {
+						id = 8870786
+						days = 141
+					}
+				}
+		}
+
+		ai_will_do = { factor = 1 }
+	}
+
+	## Filibuster state wins calls for helps
+	FIL_declare_republic = {
+		picture = new_life
+		potential = {
+			tag = FIL
+			NOT = { has_country_flag = fil_declare_republic }
+		}
+
+		allow = {
+			war = no
+			NOT = { exists = NIC }
+		}
+
+		effect = {
+			prestige = 10
+			set_country_flag = fil_declare_republic
+
+			FIL = {
+				add_country_modifier = {
+					name = declared_republic
+					duration = 1096
+				}
+				USA = {
+					country_event = {
+						id = 8870781
+						days = 14
+					}
+				}
+			}
+		}
+
+		ai_will_do = { factor = 1 }
+	}
+
+
+	## If Filibuster State exists, they can try and join the CSA even if they are a USA puppet
+	FIL_join_CSA = {
+		picture = golden_circle
+		potential = {
+			tag = FIL
+			NOT = { has_country_flag = filibuster_wars_active }
+			exists = CSA
+		}
+
+		allow = {
+			war = no
+		}
+
+		effect = {
+			FIL = {
```

Comment:
This first scope is unnecessary. FIL = { USA = { ... } } is completely equivalent to USA = { ... }